moh_ingest.py

```diff
@@ -0,0 +1,248 @@
+        or `None` if the data could not be retrieved or the file does not exist.
+
+    Examples
+    --------
+    >>> read_json("https://example.com/remote_file.json")
+    >>> read_json("data/local_file.json")
+    """
+
+    if file_path.startswith("http"):
+        try:
+            response = requests.get(file_path)
+            response.raise_for_status()
+            data = response.json()
+            return data
+        except requests.exceptions.RequestException as e:
+            print("Failed to retrieve data. Error:", e)
+            return None
+    else:
+        try:
+            with open(file_path, "r") as f:
+                data = json.load(f)
+                return data
+        except FileNotFoundError as e:
+            print("File not found. Error:", e)
+            return None
+
+
+def clean_data(katsu_server_url, headers):
+    """
+    Sends a DELETE request to the Katsu server to delete all data.
+    """
+    response = input("Are you sure you want to delete the database? (yes/no): ")
+
+    if response == "yes":
+        delete_url = "/api/v1/delete/all"
+        url = katsu_server_url + delete_url
+
+        if headers == "GET_AUTH_HEADER":
+            headers = auth.get_auth_header()
+        res = requests.delete(url, headers=headers)
+        if res.status_code == HTTPStatus.NO_CONTENT:
+            print(f"Delete successful with status code {res.status_code}")
+        else:
+            print(
+                f"Delete failed with status code {res.status_code} and message: {res.text}"
+            )
+    else:
+        print("Delete cancelled")
+        exit()
+
+
+def ingest_data(katsu_server_url, data_location, headers):
+    """
+    Send POST requests to the Katsu server to ingest data.
+    """
+    file_mapping = OrderedDict(
+        [
+            ("programs", "Program.json"),
+            ("donors", "Donor.json"),
+            ("primary_diagnoses", "PrimaryDiagnosis.json"),
+            ("specimens", "Specimen.json"),
+            ("sample_registrations", "SampleRegistration.json"),
+            ("treatments", "Treatment.json"),
+            ("chemotherapies", "Chemotherapy.json"),
+            ("hormone_therapies", "HormoneTherapy.json"),
+            ("radiations", "Radiation.json"),
+            ("immunotherapies", "Immunotherapy.json"),
+            ("surgeries", "Surgery.json"),
+            ("follow_ups", "FollowUp.json"),
+            ("biomarkers", "Biomarker.json"),
+            ("comorbidities", "Comorbidity.json"),
+        ]
+    )
+    ingest_finished = False
+    for api_name, file_name in file_mapping.items():
+        ingest_str = f"/api/v1/ingest/{api_name}"
+        ingest_url = katsu_server_url + ingest_str
+
+        print(f"Loading {file_name}...")
+        payload = read_json(data_location + file_name)
+        if payload is not None:
+            if headers == "GET_AUTH_HEADER":
+                headers = auth.get_auth_header()
+            response = requests.post(
```

Comment:
This POST request doesn't seem to work for me, as I get the error
```
(CanDIGv2) fnguyen@fnguyen-worktop:~/Documents/candigv2-ingest$ python moh_ingest.py
Select an option:
1. Run check
2. Ingest data
3. Clean data
4. Exit
Enter your choice [1-3]: 2
Loading Program.json...

REQUEST STATUS CODE: 415 
RETURN MESSAGE: {"detail":"Unsupported media type \"text/plain\" in request."}
```

Recommend adding the Content-Type header, as that made it work for me:
```suggestion
            headers["Content-Type"] = "application/json"
            response = requests.post(
```